mitielib/include/mitie.h

```diff
@@ -396,6 +413,22 @@ extern "C"
             - The returned object MUST BE FREED by a call to mitie_free().
             - If the object can't be created then this function returns NULL.
     !*/
+
+    MITIE_EXPORT mitie_text_categorizer* mitie_load_text_categorizer_pure_model (
+        const char* filename,
+        const char* fe_filename
+    );
+    /*!
+        requires
+            - filename == a valid pointer to a NULL terminated C string
+            - fe_filename == a valid pointer to a NULL terminated C string
+        ensures
+            - Reads a saved MITIE text categorizer object from disk and returns a
+              pointer to the text categorizer. filename must point to a serialised pure_model
```

Comment:
What is a pure_model?  The references to classes should use the full class name so it's clear what class is being referenced.